CMake/CodeCoverage.cmake

```diff
@@ -0,0 +1,303 @@
+# Copyright (c) 2012 - 2017, Lars Bilke
```

Comment:
Seems like a lot of copy-pasta. I suppose there's no better place to include this from.

Reply:
yeah, it's versioned in; small amount of code like this I find it better